gcp/modules/gcp-stackdriver-monitoring/resources/alert_policies/firewalls_activity.json

```diff
@@ -29,7 +29,7 @@
         "denominator_filter": "",
         "denominator_aggregations": []
       },
-      "display_name": "Log-based compute.firewalls activity check"
+      "display_name": "Log-based compute firewalls creation / modification check"
```

Comment:
Consistent with my preferred naming scheme, I might call this one something like "No GCE firewall rules are created or modified"

Reply:
https://issues.gpii.net/browse/GPII-3942